### ISSUE TITLE
Moya 10.0.1 installation via Carthage is not working

### ISSUE THREAD

**Issue by eMdOS** [Moya/Moya#1542]
Cartfile:

```
github "Moya/Moya" ~> 10.0
```

Cartfile.resolved:
```
github "Moya/Moya" "10.0.1"
```

Error:
```
*** Building scheme "Moya" in Moya.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/emdos/Development/eMdOS/Playground/Carthage/Checkouts/Moya/Moya.xcodeproj -scheme Moya -configuration Release -derivedDataPath /Users/emdos/Library/Caches/org.carthage.CarthageKit/DerivedData/9.2_9C40b/Moya/10.0.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath ./ SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO (launched in /Users/emdos/Development/eMdOS/Playground/Carthage/Checkouts/Moya)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/m7/7xddbtzx2rz52pdm6n_ybgv40000gn/T/carthage-xcodebuild.h549xk.log
```

**Comment by SD10** [comment]
@lucas34 Any input on this one? I don’t use Carthage but it looks like maybe we need to use 10.0.1 in the Cartfile?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by lucas34** [comment]
Not sure about this issue,
Could you please provide more information such as:
- The command that you ran
- Carthage Version
- Xcode Version

And also, can you please run your command with `--verbose` ?

I just made a quick test with only `github "Moya/Moya" ~> 10.0` in a cartfile. Running `carthage bootstrap` works just fine. My Cartfile is resolved with those dependencies:

Cartfile.resolved
```
github "Alamofire/Alamofire" "4.6.0"
github "Moya/Moya" "10.0.1"
github "ReactiveCocoa/ReactiveSwift" "2.0.1"
github "ReactiveX/RxSwift" "4.1.1"
github "antitypical/Result" "3.2.4"
```

**Comment by moerter** [comment]
I get the same error.

Xcode: 9.2
Carthage: 0.27.0

Running `carthage update` says
>"Incompatible Swift version - framework was built with 4.0.2 (swiftlang-900.0.69.2 clang-900.0.38) and the local version is 4.0.3 (swiftlang-900.0.74.1 clang-900.0.39.2)."

Running `carthage update --platform iOS --no-use-binaries --verbose`:
```
*** Building scheme "Moya" in Moya.xcodeproj

The following build commands failed:
	PhaseScriptExecution Swiftlint /Users/xyz/Library/Caches/org.carthage.CarthageKit/DerivedData/9.2_9C40b/Moya/10.0.1/Build/Intermediates.noindex/ArchiveIntermediates/Moya/IntermediateBuildFilesPath/Moya.build/Release-iphoneos/Moya.build/Script-46AE31F21F86DDC6004E4236.sh
(1 failure)
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/xyz/Projects/xyz/Carthage/Checkouts/Moya/Moya.xcodeproj -scheme Moya -configuration Release -derivedDataPath /Users/xyz/Library/Caches/org.carthage.CarthageKit/DerivedData/9.2_9C40b/Moya/10.0.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/dm/0d2chvhn6s57mkvwzhbdgc240000gq/T/Moya SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO (launched in /Users/xyz/Projects/xyz/Carthage/Checkouts/Moya)
```

Cartfile.resolved
```
github "Alamofire/Alamofire" "4.6.0"
github "Moya/Moya" "10.0.1"
github "ReactiveCocoa/ReactiveSwift" "2.0.1"
github "ReactiveX/RxSwift" "4.1.1"
github "antitypical/Result" "3.2.4"
```

## SOLUTION:
Installing SwiftLint  and linking it solved the problem and Moya build succeeded
```
brew install swiftlint
brew link swiftlint
```

**Comment by AndrewSB** [comment]
Oh no :/

**Comment by lucas34** [comment]
I uninstall Swiftlint and I'm still able to fetch Moya (11.0) properly with Carthage.

**Comment by shwetachitlangia** [comment]
Any update when this will be resolved?

**Comment by SD10** [comment]
@shwetachitlangia Are you facing the same error regarding Swiftlint? @lucas34 was apparently able to fetch Moya without SwiftLint installed. Would love some feedback on this from Moya Carthage users

**Comment by SD10** [comment]
I just released Moya 11.0.2 with the patch for this. Please re-open if the problem continues
